.github/workflows/dbc.yaml

```diff
@@ -0,0 +1,36 @@
+name: update dbc json
+on:
```

Comment:
have you tested this? should be able to test by adding a `push` trigger

Reply:
tested it locally by running the same docker image against the script, but I can try this out